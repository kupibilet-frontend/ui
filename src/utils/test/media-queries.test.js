import { css } from 'styled-components'
import media from '../media-queries'

test('media-queries', () => {
  const MediaSnap = css`
    .class {
      ${media.mobile`
        background: black;

        &:hover {
          background: white;
        }
      `}

      ${media.tablet`
        background: black;

        &:hover {
          background: white;
        }
      `}

      ${media.desktop`
        background: black;

        &:hover {
          background: white;
        }
      `}
    }
  `

  expect(MediaSnap).toMatchSnapshot()
})
