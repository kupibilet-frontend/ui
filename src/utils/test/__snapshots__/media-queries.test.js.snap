// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`media-queries 1`] = `
Array [
  "
    .class {
      ",
  "
    .responsive &, .responsive-booking & {
      @media ",
  "screen and (max-width: 599px)",
  " {
        ",
  "
        background: black;

        &:hover {
          background: white;
        }
      ",
  "
      }
    }
  ",
  "

      ",
  "
    .responsive &, .responsive-booking & {
      @media ",
  "screen and (min-width: 600px) and (max-width: 999px)",
  " {
        ",
  "
        background: black;

        &:hover {
          background: white;
        }
      ",
  "
      }
    }
  ",
  "

      ",
  "
    .responsive &, .responsive-booking & {
      @media ",
  "screen and (min-width: 1000px)",
  " {
        ",
  "
        background: black;

        &:hover {
          background: white;
        }
      ",
  "
      }
    }
  ",
  "
    }
  ",
]
`;
